src/screens/HyperSwitch/Customers/CustomersEntity.res

```diff
@@ -0,0 +1,96 @@
+open LogicUtils
+
+type customers = {
```

Comment:
Can we move all the types in the Entity file to a new CustomerType File ?

Reply:
Updated